pkgs/top-level/haskell-packages.nix

```diff
@@ -1,6 +1,16 @@
 { pkgs, callPackage, stdenv, buildPlatform, targetPlatform }:
 
-rec {
+let # These are the GHC versions that support building with integer-simple.
```

Comment:
This list contains every compiler we have except 7.0.4, I believe. Assuming that newer versions released in the future will probably support this feature, too, would it make sense to invert this logic, i.e. to list those compilers that *don't* support a `gmp`-less build? That would make this list shorter and it would remove the need to do manual maintenance in the future because new compiler versions are implicitly supported the moment we add them.